docs/nlu/training-data-format.rst

```diff
@@ -15,10 +15,100 @@ Data Formats
 ~~~~~~~~~~~~
 
 
-You can provide training data as Markdown or as JSON, as a single file or as a directory containing multiple files.
-Note that Markdown is usually easier to work with.
+You can provide training data as YAML, Markdown or as JSON, as a single file or as a directory containing multiple files.
 
 
+YAML Format
```

Comment:
should we really split this up? @akelad I think it might make more sense to have a single page that contains all the information about the training data format, ala:
1. general structure (it is yaml, multi file, what are the keys)
2. story training data
3. intent training data
...

Reply:
I'd vote for one page.